common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -447,7 +447,10 @@ workflow_state = ? ` +
 	templateUpdateWorkflowExecutionConditionSuffix = ` IF next_event_id = ?`
 
 	templateUpdateWorkflowExecutionQuery = `UPDATE executions ` +
```

Comment:
let us create a dedicated query just for version history.
for forward / backward compatibility, create an issue.

Reply:
Done